### ISSUE TITLE
Focus ring on Narrators table remove buttons is clipped

### ISSUE THREAD

**Issue by mikiher** [audiobookshelf/audiobookshelf-client-react#159]
**Repro**
1. Go to Narrators page
2. Navigate to one of the remove buttons using the keyabord

**Expected**: the focus ring should be fully visible.
**Actual:**
<img width="200" height="93" alt="Image" src="https://github.com/user-attachments/assets/a95ed466-e61d-437a-ab15-b443d0c15a0b" />

**Comment by Div-47** [comment]
Hi @mikiher  I’d like to work on this issue. Could you please assign it to me? I have experience with React and frontend UI fixes, and this looks like a beginner–moderate task that I’d be happy to take on. Thanks!
